packages/paste-core/primitives/box/src/index.tsx

```diff
@@ -159,6 +161,7 @@ const extraConfig = system({
   willChange: true,
   textDecoration: true,
   textTransform: true,
+  textAlign: true,
```

Comment:
Does this need to be added to `safelySpreadBoxProps` (I realize there maybe more missing)

Reply:
@SiTaggart I added `TYPOGRAPHY_PROPS` to `safelySpreadBoxProps`. Does that cover it, or do I need to call out `textAlign` specifically?

Reply:
Dunno, you'd have to check whats in the typography props. if we list it in there then it's enough, if we don't then it needs to be a special prop we add on it's own to the object

Reply:
OK, yeah it's listed in the props `https://github.com/twilio-labs/paste/blob/main/packages/paste-style-props/src/typography.ts#L8